### ISSUE TITLE
Feedback on Tutorial: Build an SAP Fiori UX app with SAP Gateway data

### ISSUE THREAD

**Issue by qmacro** [SAPDocuments/Tutorials#1155]
In Step 10 "Specify data for the "detail" view", the student is instructed to leave the value blank, for the Line Item Numeric Attribute, but specify a value for Line Item Unit Measure (a binding to the `CountryText` property). 

This is ultimately setting bindings for an Object Header control. But the `number` and `numberUnit` properties in this control, which tie up directly with the two parameters above, are designed so that the `numberUnit` will only be shown if there's a value for the `number` property. 

In other words, the Country Text won't show up in the Detail, because there's nothing in the numeric attribute. This is confirmed in the screenshot in Step 18.

**Comment by akula86** [comment]
Updated tutorial: Commit: https://github.com/SAPDocuments/Tutorials/commit/f4b5591e59fa886d3a66050759facebb012025f1
